language-design.md

```diff
@@ -26,6 +38,10 @@ you know a poorly designed language? What are the symptoms?
 
 **Response**
 
+I would know a language is well-designed if it is "humanely designed", following the description of Pavlus, meaning that it is sufficiently readable such that a novice programmer could look at it and interpret what it does. Additionally, it should have good documentation so that people can learn about it without having to manually deconstruct code or having to use the language, as detailed in Bloch's piece. 
```

Comment:
I like your perspective on this! My take was very similar; I argued that well-designed languages are intuitive enough for new users but robust enough for experienced ones. At least that was my take, but my critique partner convinced me otherwise on needing to cater to power users, as that would mean that things like Scratch aren’t well-designed, and I think it is!